src/main/java/org/mule/module/apikit/helpers/AttributesHelper.java

```diff
@@ -137,9 +137,14 @@ public static String getContentType(MultiMap<String, String> headers) throws Uns
    * @param mediaType Media Type
    * @return Media Type with "type/subtype" format
    */
-  public static String getMediaType(String mediaType) {
-    MediaType mType = parse(mediaType);
-    return String.format("%s/%s", mType.getPrimaryType(), mType.getSubType());
+  public static String getMediaType(String mediaType) throws UnsupportedMediaTypeException {
+    try {
+      MediaType mType = parse(mediaType);
+      return String.format("%s/%s", mType.getPrimaryType(), mType.getSubType());
+    } catch (Exception e) {
+      String message = mediaType == null ? "MediaType is null" : e.getMessage();
```

Comment:
I guess this message replaces a NPE, which it doesn't have much sense to me

Reply:
It could happen that request contains "Content-Type: " exactly like that and throws NPE, this is why I thought of handling that case to show unsupported media type message.